libobs/obs-source-deinterlace.c

```diff
@@ -418,8 +419,11 @@ void deinterlace_render(obs_source_t *s)
 			tech_name = "DrawMultiply";
 			multiplier = obs_get_video_sdr_white_level() / 80.0f;
 		}
+		PRAGMA_WARN_POP
```

Comment:
I think this pragma can put outside of the upper `switch` statement instead of having for each.

Reply:
MSVC pragma emits a message because MSVC doesn't know "warning without erroring" and so I want/need one per switch.